src/es5/rxjs-testing.js

```diff
@@ -0,0 +1 @@
+export * from "rxjs/_esm5/testing/index";
```

Comment:
I'm not really sure this is necessary to expose but I figured because I was here anyways I'd do it